Views/Order/DisplayOrders.cshtml

```diff
@@ -0,0 +1,59 @@
+﻿@model PizzaOrdering_Mvc.Models.Pizza
+@{
```

Comment:
Create and use OrderViewModel instead of showing some properties in ViewBag and dictionary.